rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -269,10 +269,10 @@ $$("#${detailsId}").bind( "show", function(event, ui) {
     <li><a href={htmlId_#(jsId,"sd_fs_")}>File systems</a></li>
     <li><a href={htmlId_#(jsId,"sd_net_")}>Network interfaces</a></li>
     <li><a href={htmlId_#(jsId,"sd_soft_")}>Software</a></li>
-    <li><a href={htmlId_#(jsId,"sd_props_")}>Properties</a></li>
-    <li><a href={htmlId_#(jsId,"sd_var_")}>Environment variables</a></li>
+    <li><a href={htmlId_#(jsId,"sd_var_")}>Environment</a></li>
     <li><a href={htmlId_#(jsId,"sd_process_")}>Processes</a></li>
     <li><a href={htmlId_#(jsId,"sd_vm_")}>Virtual machines</a></li>
+    <li><a href={htmlId_#(jsId,"sd_props_")}>Properties</a></li>
```

Comment:
Note that this change doesn't fix the English, it moves the "Properties" tab to the end of the list, to avoid it being "lost" in the middle of inventory data (which it is not).